modules/nf-core/taxpasta/standardise/main.nf

```diff
@@ -0,0 +1,36 @@
+process TAXPASTA_STANDARDISE {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "bioconda::taxpasta=0.1.1"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/taxpasta:0.1.1--pyhdfd78af_0':
+        'quay.io/biocontainers/taxpasta:0.1.1--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(profile)
+    path taxonomy
+
+    output:
+    tuple val(meta), path("*.{tsv,csv,arrow,parquet,biom}"), emit: profiles
```

Comment:
```suggestion
    tuple val(meta), path("*.{tsv,csv,arrow,parquet,biom}"), emit: standardised_profile
```

I think that's clearer, you also only produce a single profile right?

Reply:
Yes, only one. I always name my channels in plural 🤷🏼‍♂️ but this makes sense, too.